Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/bootstrap.py

```diff
@@ -394,6 +402,9 @@ def bootstrap_AssetBuilder():
         elif O3DE_EDITOR.stem.lower() == "materialcanvas":
             _settings = bootstrap_MaterialEditor()
 
+        elif O3DE_EDITOR.stem.lower() == "passcanvas":
+            _settings = bootstrap_MaterialEditor()
+
```

Comment:
Copy and past error, wrong needs to be `bootstrap_PassCanvas`

Reply:
I don't think it's an error. I'll have to confirm with Jonny. There are multiple instances of bootstrap_MaterialEditor(), which I think doesn't do anything.

Reply:
This PR adds `bootstrap_PassCanvas`